components/Forms/TransactionForm.js

```diff
@@ -5,26 +5,60 @@ export default function TransactionForm({
   onAddTransaction,
   formType,
   onCloseModal,
+  savingsTransferSum,
 }) {
   function handleSubmit(event) {
     event.preventDefault();
+

```

Comment:
**Just a suggestion**
The repetitive use of `formatDate(new Date())` can be optimised by setting a default date value if `data.date` is `falsy`.

```js
const transactionDate = data.date ? formatDate(data.date) : formatDate(new Date());
```